### PR TITLE
New look!

### PR DESCRIPTION
**Pull request by jodiedoubleday**
The keep with Holiday Extras new employer brand colours, the hungry geek blog has had a bit of a redesign.

__Extras__
- Changed the speakers page to be less complex
- Removed people who no longer work for HX from the team page
- Remove speaker events
- Added conference logos to speaker pages.

__To Test__
- Run the blog, check a few pages and ensure the colours work well

screen shot 2015-09-25 at 14 25 33

### CODE REVIEW

File: _includes/speaking_calendar.html [hunk 1]
```diff
@@ -1,51 +1,112 @@
...
-        {% endfor %}
-      </ul>
-    </li>
+        </div>
+      </li>
     {% endfor %}
   </ul>
-  {% endfor %}
-  {% endif %}
+<!--
```
Comment (lukefrake) [inline _includes/speaking_calendar.html]: Can we delete this? It will be in version anyway.

File: _includes/speaker_list.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{% if speakers.size %}
+<!-- {% if speakers.size %}
```
Comment (lukefrake) [inline _includes/speaker_list.html]: Can this go as it's in version history?

File: less/speaking.less [hunk 3]
```diff
@@ -5,10 +5,36 @@
   }
 
   h2 {
-    text-align: center;
+    // text-align: center;
     font-size: 2.2em;
   }
+  ul {
+    list-style:none;
+    paddong:0;
```
Comment (lukefrake) [inline less/speaking.less]: typo `paddong`

paddongadong

File: less/speaking.less [hunk 4]
```diff
@@ -5,10 +5,36 @@
   }
 
   h2 {
-    text-align: center;
+    // text-align: center;
```
Comment (lukefrake) [inline less/speaking.less]: Can take this out

File: less/speaking.less [hunk 5]
```diff
@@ -5,10 +5,36 @@
...
+}
 
+.event-logo {
+  float:left;
+  width:28%;
+  padding-right:2%;
+  img {
+    width:100%;
+  }
+  & + div {
```
Comment (lukefrake) [inline less/speaking.less]: don't think you need `&` as you have a space

File: _data/speaking.yml [hunk 6]
```diff
@@ -1,56 +1,63 @@
...
+- name: 'FullStack 2015'
+  logo: 'fullstack.png'
+  location: 'London, UK'
+  link: 'https://skillsmatter.com/conferences/6612-fullstack'
+  pretty_date: '26th - 28th October, 2015'
+  date_for_ordering: '2015-10-26'
+  speaker: 'Oliver Rumbelow'
+  talk: 'Blowing Out The LAMP'
+
+# - name: 'TEDx University of Kent'
```
Comment (lukefrake) [inline _data/speaking.yml]: Could drop all these as well?

**Comment by lukefrake** [comment]
Looks really nice, couple of comments on the code but ready to :+1: after that!

File: _data/speaking.yml (hunk 6 above)
Comment (jodiedoubleday) [inline _data/speaking.yml]: Im gonna keep these in there commented, mainly because in a future PL I want to do "past events"

Comment (lukefrake) [inline _data/speaking.yml]: Sounds good :+1:

**Comment by lukefrake** [comment]
:shipit:

**Pull request merged**